### PR TITLE
PetHRRT pipeline fixes (v4)

### PR DESCRIPTION
**Pull request by laemtl**
Last changes to fix:
- wrong status message
- acquisition date

### CODE REVIEW

File: uploadNeuroDB/NeuroDB/ImagingUpload.pm [hunk 1]
```diff
@@ -363,8 +363,11 @@ sub runHrrtInsertion {
     $command .= " -verbose " if ($this->{'verbose'});
 
     my $output = $this->runCommandWithExitCode($command);
-
-    return $output;
+    
+    if ( $output == 0 ) {
```
Comment (laemtl) [inline uploadNeuroDB/NeuroDB/ImagingUpload.pm:367]: error detection is made with
`if ( !$output ) {` and success code (0) is a falsy value.

File: uploadNeuroDB/NeuroDB/MRI.pm [hunk 2]
```diff
@@ -780,7 +780,7 @@ sub register_db {
         'FileType',        'InsertedByUserID', 'Caveat',
         'SeriesUID',       'TarchiveSource',   'HrrtArchiveID',
         'SourcePipeline',  'PipelineDate',     'SourceFileID',
-        'ScannerID'
+        'ScannerID',       'AcquisitionDate'
```
Comment (laemtl) [inline uploadNeuroDB/NeuroDB/MRI.pm:783]: Required to save the value stored in the file class to the files table.

File: uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm [hunk 3]
```diff
@@ -1296,6 +1296,8 @@ sub registerScanIntoDB {
             || $study_start_date
             || undef;
 
+        print "Acquisition date: " . $acquisition_date . "\n";
```
Comment (laemtl) [inline uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm:1299]: Can be left here if we want some hints during the pipeline execution.

**Review by cmadjar — APPROVED** [review]
LGTM

**Pull request merged**